R/helpers.R

```diff
@@ -103,16 +103,25 @@ setMethod(
     studyArea <- studyArea[, -c(1:ncol(studyArea))]
     studyArea <- as(studyArea, "SpatialPolygons")
     studyAreaName(studyArea, ...)
-})
+  })
 
 #' @export
 #' @rdname studyAreaName
 setMethod(
   "studyAreaName",
-  signature = "SpatialPolygons",
+  signature = "ANY",
```

Comment:
why `ANY` and not `spatialClasses`, which includes `sf` objects already?

Reply:
because `sf` objects are not in `spatialClasses`. @eliotmcintire checked

Reply:
I was unable to include `sf` class objects in `spatialClasses`... the class generic, apparently, is not exported from `sf`

Reply:
Also, the actual content of the function is simple: `digest(theGeometryOfTheStudyArea)` ... so that applies to any object, so we don't need a specific signature for specific classes.

Reply:
ok, thanks for clarifying